scripts/templates/values.yaml

```diff
@@ -154,3 +154,8 @@ rules:
     ## This will add another condition to negate the dependecy like:
     ## when(signal > {{severity}}) and when(signal <= {{severity.dependency}})
     #dependency:
+
+    ## @param unit - string - optional
+    ## Threshold unit for the current detector.
+    ## This will update the detector message and associated variables description
+    unit:
```

Comment:
I am not sure it is the good place to put it.
Define it at detector level will be easier to configure and more suited to our model where one detector is atomic and only contains 1 "check"